### ISSUE TITLE
Template set internal notes are not being displayed? 

### ISSUE THREAD

**Issue by Xm4Tr1X** [osTicket/osTicket#3796]
Installation details:
osTicket Version	v1.10 (901e5ea) —  Up to date
Web Server Software	Apache/2.4.12 (Win32) OpenSSL/1.0.1l PHP/5.6.8
MySQL Version	5.6.24
PHP Version	5.6.8
### Description

Internal notes posted on template group set are not displayed.
fantastic notes and where to find them?(sorry i couldnt help my self)

### Steps to Reproduce

 Adminpanel->Panel-> Email->templates-> select any template group and post an internal note 

**Expected behavior:** [What you expected to happen]
The posted note should be displayed
**Actual behavior:** [What actually happened]
The notes posted are saved in the db but are not displayed.

TL:DR  Display them notes please

There seems to be an old issue that had been closed without resolving it ?
#486

**Comment by ntozier** [comment]
Image showing what you are talking about?
Because right now I have no idea what you are talking about.

**Comment by GodOfWarWeed** [comment]
ok here's the screen shot. 
![templatebug](https://cloud.githubusercontent.com/assets/19608968/25090277/e7504f54-239f-11e7-8c4c-eae01375fafa.png)

For more clarification, why dont you take a look at #486.

**Comment by CarlosMoraRoa** [comment]
This is because the variable $info[] is overwrite when osticket do the foreach to show the templates, a simple fix is copy $info[] in another array to keep the the tpl info.

**Comment by killerbees19** [comment]
@JediKev This is still an issue! ;-)

Just tested PR #3901 at develop/1.14.x branch – works like a charm now.

**Comment by JediKev** [comment]
@killerbees19 

Thank you, we will look into this and get back to you.

Cheers.

**Comment by JediKev** [comment]
@killerbees19 

Here is the official pull request:
- https://github.com/osTicket/osTicket/pull/5580

I didn't like how the original PR set yet another variable instead of preventing the variable overwrite in the first place. This makes it to where the original `$info` variable is not overwritten.

Cheers.
